src/Entities/Regular/RegularSelection.java

```diff
@@ -9,30 +9,37 @@
 public class RegularSelection implements ISelection {
     HashMap<IAddon, Integer> singleSingleton;
 
+    /**
+     * Creates a RegularSelection object
+     * @param selection the addons a customer has chosen for a singleton
+     */
     public RegularSelection(HashMap<IAddon, Integer> selection){
         this.singleSingleton = selection;
     }
 
-    public float getPrice(){
```

Comment:
I think selection should have a price function so we can get the price for the cart without going into the nested list and calculating it manually there

Reply:
right